esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -1666,8 +1660,7 @@ bool CwssqlEx::cloneAndExecuteWU(IEspContext &context, const char * originalwuid
 
 bool CwssqlEx::onCreateTableAndLoad(IEspContext &context, IEspCreateTableAndLoadRequest &req, IEspCreateTableAndLoadResponse &resp)
 {
-    if (!context.validateFeatureAccess(WSSQLACCESS, SecAccess_Write, false))
-            throw MakeStringException(-1, "Failed to fetch results (open workunit). Permission denied.");
+    context.ensureFeatureAccess(WSSQLACCESS, SecAccess_Write, -1, "Failed to fetch results (open workunit). Permission denied.");
```

Comment:
original error message was wrong. should be "CwssqlEx::onCreateTableAndLoad: Permission denied."